static/app/views/discover/table/tableView.tsx

```diff
@@ -646,6 +646,16 @@ export const StyledLink = styled(Link)`
   }
 `;
 
+export const TransactionLink = styled(Link)`
+  & div {
+    display: block;
+    overflow: hidden;
+    max-width: 100%;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
```

Comment:
```suggestion
  ${p => p.theme.overflowEllipsis}
```

We have this theme for ellipsis overflow that works most of the time! It seems like it works here without having to select the div